walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -75,25 +75,25 @@
       TimeRange proposedSlot = TimeRange.fromStartEnd(TimeRange.START_OF_DAY, times.get(0).start(), false);
       addIfProposedTimeIsLongEnough(proposedSlot, request, complementTimes);
     }
-    int i = 0;
+    int current = 0;
     int next = 1;
     while (next < times.size()) {
       // If there is any space between the current time and the next time, add it to the list
       // Otherwise, keep current time the same until the "next" event is one not entirely contained in the current one 
-      if (!times.get(i).overlaps(times.get(next))) {
-        TimeRange proposedSlot = TimeRange.fromStartEnd(times.get(i).end(), times.get(next).start(), false);
+      if (!times.get(current).overlaps(times.get(next))) {
+        TimeRange proposedSlot = TimeRange.fromStartEnd(times.get(current).end(), times.get(next).start(), false);
         addIfProposedTimeIsLongEnough(proposedSlot, request, complementTimes);
       }
-    	if (! times.get(i).contains(times.get(next))) {
-        i++;
+    	if (!times.get(current).contains(times.get(next))) {
```

Comment:
I wonder if it might be clearer to do:
```
if (contains) {
  next++;
} else {
  current++;
  next = current+2;
}
```
I guess im confused by why it's current+2 instead of current+1

Reply:
I think it's because lines 88 and 90 should actually be swapped. I just tried switching them, and my tests still pass, which is concerning. I had better go write some more tests...

Reply:
It turns out the real solution was actually much simpler than any of this. Oops.

Reply:
This is so much simpler!